.travis.yml

```diff
@@ -3,5 +3,6 @@ git:
   depth: 2
 language: node_js
 node_js:
+  - 'iojs'
```

Comment:
That one doesn't actually look quite consistent. `'0.12'` and `'0.10'` are quoted but `iojs` is not. Would you consider changing that one? If not, I'll fix this one soon.